### PR TITLE
Translate Using server-sent events into korean

### PR DESCRIPTION
**Pull request by uyeong**

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Sorry, but this locale is currently frozen and changes/fixes are not currently accepted. Please see https://hacks.mozilla.org/2020/12/an-update-on-mdn-web-docs-localization-strategy/ and [README](https://github.com/mdn/translated-content/blob/main/README.md) of this repo.

**Comment by SphinxKnight** [comment]
@uyeong Please check #519 if you would like to participate in a possible Korean localization effort to get this locale "unfrozen" :)

File: files/ko/web/api/server-sent_events/using_server-sent_events/index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+- Server-sent events
+- messaging
+---
+<p>{{DefaultAPISidebar("Server Sent Events")}}</p>
+
+<div class="summary">
+    <p><a href="/ko/docs/Web/API/Server-sent_events">Server-Sent Events</a>를 사용하는 웹 애플리케이션 개발은 매우 간단하다. 웹 애플리케이션으로 스트림 이벤트를 보내는 약간의 코드가 서버 상에 필요하지만, 웹 애플리케이션 측은 <a href="/en-US/docs/Web/API/WebSockets_API">웹 소켓</a>에서 이벤트를 다루는 방식과 거의 차이가 없다.</p>
+</div>
+
+<h2 id="Receiving_events_from_the_server" name="Receiving_events_from_the_server">サーバからイベントを受信する</h2>
```
Comment (hochan222) [inline files/ko/web/api/server-sent_events/using_server-sent_events/index.html:20]:
```suggestion
<h2 id="Receiving_events_from_the_server" name="Receiving_events_from_the_server">서버에서 이벤트 받기</h2>
```

File: files/ko/web/api/server-sent_events/using_server-sent_events/index.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+<pre>event: userconnect
+data: {"username": "bobby", "time": "02:33:48"}
+
+data: Here's a system message of some kind that will get used
+data: to accomplish some task.
+
+event: usermessage
+data: {"username": "bobby", "time": "02:34:11", "text": "Hi everyone."}</pre>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
```
Comment (hochan222) [inline files/ko/web/api/server-sent_events/using_server-sent_events/index.html:186]:
```suggestion
<h2 id="Browser_compatibility">브라우저 호환성</h2>
```

**Review by hochan222 — CHANGES_REQUESTED** [review]
동결이 해제됨에 따라 리뷰를 진행했습니다. 몇가지 단어들에 대해 한국어로 대체되면 더 좋을듯해서 리뷰했습니다. 기여해주셔서 감사합니다 좋은 하루 보내세요 :)

**Review by hochan222 — APPROVED** [review]
LGTM :)

**Pull request merged**